### PR TITLE
Linking my blog to the gci fossasia website

### PR DESCRIPTION
**Pull request by hshafaq7**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [ ] Read and understood (see CONTRIBUTING.md)
- [ ] Included a Preview link and screenshot showning after and before the changes.
- [ ] Images are `240 x 240` [w x h].
- [ ] Included a description of change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<Issue no.>`

### CODE REVIEW

**Review by nikit19 — CHANGES_REQUESTED** [review]
Please upload the image. And you can make changes in the same pull request too

File: _data/blogs.yml [hunk 1]
```diff
@@ -222,3 +222,8 @@
   title: Flying with Fossasia
   img: arjun.jpg
   author_name: Arjun Pandey
+
+-url: https://mygeekyworld786.wordpress.com/2017/12/30/here-i-come-coding-in/
+ title: Here I come Coding In
+ img: hshafaq1
```
Comment (0x48piraj) [inline _data/blogs.yml:228]: wrong syntax

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
Image syntax is wrong, as mentioned in my code review.

File: _data/blogs.yml [hunk 2]
```diff
@@ -222,3 +222,8 @@
   title: Flying with Fossasia
   img: arjun.jpg
   author_name: Arjun Pandey
+
+-url: https://mygeekyworld786.wordpress.com/2017/12/30/here-i-come-coding-in/
```
Comment (abishekvashok) [inline _data/blogs.yml:226]: Add a space after `-`

**Review by abishekvashok — CHANGES_REQUESTED** [review]

**Comment by harshit98** [comment]
Closing this PR. Similar to #704

**Pull request closed without merging**